pkg/commands/report.go

```diff
@@ -62,7 +62,7 @@ func outputCheckPermsAndExit(output string) {
 	ch.Separator()
 	ch.Error("Unable to prepare the reports directory for writing the migration reports. " +
 		"Please ensure your file permissions allow the following:")
-	ch.Bullet(fmt.Sprintf("creation of the output directory if it doesn't already exist " +
+	ch.Bullet(fmt.Sprintf("creation of the output directory if it doesn't already exist "+
```

Comment:
Random change?

Reply:
while checking `static-checks` there was a failure related to fmt, so fixed it.